Makefile.work

```diff
@@ -193,6 +195,7 @@ endif
 DOCKER_RUN := docker run --rm=true --privileged --init \
     -v $(DOCKER_BUILDER_MOUNT) \
     -v "$(DOCKER_LOCKDIR):$(DOCKER_LOCKDIR)" \
+    -v $(DOCKER_ROOT):/var/lib/docker \
```

Comment:
is there way to cleanup this folder? I guess it would be better if we do not reuse the host docker folder for this inside docker builds.